indico/modules/events/registration/controllers/__init__.py

```diff
@@ -40,21 +46,32 @@ def _process_POST(self):
         if self.management:
             session['registration_notify_user_default'] = notify_user
         modify_registration(self.registration, form_data, management=self.management, notify_user=notify_user)
-        return redirect(self.success_url)
+        return jsonify({'redirect': self.success_url})
 
     def _process_GET(self):
-        registration_data = {r.field_data.field.html_field_name: camelize_keys(r.user_data)
-                             for r in self.registration.data}
+        form_data = get_flat_section_submission_data(self.regform)
         section_data = camelize_keys(get_event_section_data(self.regform, management=self.management,
                                                             registration=self.registration))
+        registration_data = {r.field_data.field.html_field_name: camelize_keys(r.user_data)
+                             for r in self.registration.data}
+        file_data = self._get_file_data()
 
         registration_metadata = {
             'paid': self.registration.is_paid,
             'manager': self.management
         }
 
+        user_data = {t.name: getattr(session.user, t.name, None) if session.user else '' for t in PersonalDataType}
+        user_data['title'] = get_title_uuid(self.regform, user_data['title'])
         return self.view_class.render_template(self.template_file, self.event,
                                                sections=section_data, regform=self.regform,
```

Comment:
maybe you can rename this to `angular_sections` like i did in the other places as well so it's clear it's something to remove eventually?

Reply:
I'm actually using the user data to filter out the `initialValues` in `RegistrationFormSubmission`. The reason for that is that we have `unknown = RAISE` in the dynamically generated MM schema. If we eventually change that, we can also get rid of the `initialValues` filtering and also the user data

Reply:
I looked at it again and noticed that we don't use the `user_data` at all in editing (neither angular, nor react) so I removed it